### PR TITLE
Add StatusClient with URI /status/service

### PR DESCRIPTION
**Pull request by denwitham**
Adding find, find_one and list for status URIs (currently only /status/service added, but should work for /status/host etc)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,16 +94,16 @@ for all unhandled checks
...
+    acknowldege_details = client.acknowledge_event(params={'hst.hostname': 'Opsview'}, data={'notify': str(int(bool(True))), 'sticky': str(int(bool(True))), 'comment': 'Acknowledged and actioned by user'})
 
 for adding an acknowldegement to all unhandled host and service events for Opsview
-    acknowldege_details = client.acknowledge_event(params={'svc.hostname': 'Opsview', data={'notify': str(int(bool(True))), 'sticky': str(int(bool(True))), 'comment': 'Acknowledged and actioned by user'})
+    acknowldege_details = client.acknowledge_event(params={'svc.hostname': 'Opsview'}, data={'notify': str(int(bool(True))), 'sticky': str(int(bool(True))), 'comment': 'Acknowledged and actioned by user'})
 
 for deleting the acknowledgement for Opsview against check Connectivity - LAN
-    acknowledge_details = client.acknowledge_delete(params={'svc.hostname': 'Opsview', 'svc.servicename': 'Connectivity - LAN'})
+    acknowledge_details = client.acknowledge_delete(params={'svc.hostname': 'Opsview'}, 'svc.servicename': 'Connectivity - LAN'})
```
Comment (duncs) [inline README.md]: This doesn't look right - there are unbalanced `}` on the line

Comment (duncs) [reply README.md]: Ah, I think there are multiple commits all touching the same thing

**Comment by denwitham** [comment]
Hi Duncan,
Yes, sorry about that,  I didn't see my mistake in the readme until I had
committed it.
Am happy to delete the pull request and resubmit it in a single commit.

Denise

File: README.md (hunk 1 above)
Comment (denwitham) [reply README.md]: Hi Duncan,
Yes, sorry about that,  I didn't see my mistake in the readme until I had
committed it.
Am happy to delete the pull request and resubmit it in a single commit.
Just let me know if this is wanted

Denise

**Pull request closed without merging**